SpineTests/SpineTests.swift

```diff
@@ -53,7 +53,7 @@ class FindAllTests: SpineTests {
 		}
 		
 		waitForExpectations(timeout: 10) { error in
-			XCTAssertNil(error, "\(error)")
+      XCTAssertNil(error, "\(String(describing: error))")
```

Comment:
maybe you can remove `"\()"`, just use `String(describing: error)`

Reply:
Yeah, pretty sure that'll work as well. There are quite a few of these `String(describing:)` changes that I just changed via the auto migrator.